highfidelity/hifi-content
# Checkout zone

## QA Test
https://highfidelity.testrail.net/index.php?/tests/view/7031&group_by=cases:section_id&group_order=asc&group_id=323

Use hifi://dev-content/testing for the test and use the most left check-out zone, that one contains the scripts of this PR.
